db/migrate/20220808064045_create_request_log.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRequestLog < ActiveRecord::Migration[6.0]
+  def change
+    create_table :request_logs do |t|
+      t.string :log_message
```

Comment:
As mentioned above, `log_` is redundant here, so make this `message` only.

Reply:
done.